src/stories/theme/_list.css

```diff
@@ -0,0 +1,33 @@
+.ck-content ul {
+  margin: 16px 0px 16px 20px;
```

Comment:
Garden's list `margin-left` is 24px. Garden's list `<li>`s are 24px in height – 20px line height + 4px padding. I realize you are locked into a semantic ul/ol -> li structure here, but options should be explored (`::before`/`::after`?) for getting the spacing right.

Reply:
Almost all of the content needs some work when it comes to margin and relative spacing. Might be worthwhile to handle that all together.